silence/silence.go

```diff
@@ -179,6 +180,6 @@ func (sr *SilenceRequest) WillExpireBy(expiryPeriod time.Duration) (bool, error)
 }
 
 // Active returns a bool if the silence is Active
-func (sr *SilenceRequest) Active() bool {
-	return sr.Status.State == "active"
+func (sr *Request) Active() bool {
+	return sr.ID != "" && sr.Status.State == "active"
```

Comment:
Did you have an instance where the ID was `""` but the silence was active? Or just being careful?

Reply:
Really I was just making sure I didn't actually change the existing logic. I didn't think too hard about whether `sr.Status.State` could throw a NPE on a freshly-initialized `Request` struct. (Thinking it through now, I guess it can't, because `Status` isn't a pointer, so the whole thing gets go nil values?)